ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -36,6 +36,7 @@
       data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
       action="${container.metadata.action}"
       id="${container.id}"
+      data-cmp-auto-save="${wcmmode.disabled && container.autoSaveConfig && container.autoSaveConfig.enableAutoSave ? 'true' : 'false'}"
```

Comment:
This should also be added in the form container read me. All data attributes should be part of the read me file, so that customers who have extended form container component are aware of the same

In the read me, also add the json property details and why this data attribute was created